web/docs/database/backends.md

```diff
@@ -1,3 +1,329 @@
+
+Your Wasp app will automatically connect to it, just keep `wasp start db` running in the background.
+Also, make sure that:
+
+- You have Docker installed and in `PATH`.
+- The port `5432` isn't taken.
+
+### Connecting to an existing database
+
+If, instead of using `wasp start db`, you'd rather spin up your own dev database (or connect to an external one), you'll need to tell Wasp about it using the `DATABASE_URL` environment variable. Wasp will use the value of `DATABASE_URL` as a connection string.
+
+The easiest way to set the necessary `DATABASE_URL` environment variable is by adding it to the [.env.server](/docs/project/dotenv.md) file in the root dir of your Wasp project (if that file doesn't yet exist, create it).
+
+Alternatively, you can set it inline when running `wasp` (this applies to all environment variables):
+
+```bash
+DATABASE_URL=<my-db-url> wasp ...
+```
+
+This trick is useful for running a certain `wasp` command on a specific database.
+For example, you could do:
+
+```bash
+DATABASE_URL=<production-db-url> wasp db seed myProductionSeed
+```
+
+This command seeds the data for a fresh staging or production database.
+To more precisely understand how seeding works, keep reading.
+
+## Seeding the Database
+
+**Database seeding** is a term used for populating the database with some initial data.
+
+Seeding is most commonly used for two following scenarios:
+
+1.  To put the development database into a state convenient for working and testing.
+2.  To initialize any database (`dev`, `staging`, or `prod`) with essential data it requires to operate.
+    For example, populating the Currency table with default currencies, or the Country table with all available countries.
+
+### Writing a Seed Function
+
+You can define as many **seed functions** as you want in an array under the `app.db.seeds` field:
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```wasp title=main.wasp
+app MyApp {
+  // ...
+  db: {
+    // ...
+    seeds: [
+      import { devSeedSimple } from "@server/dbSeeds.js",
+      import { prodSeed } from "@server/dbSeeds.js"
+    ]
+  }
+}
+```
+
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```wasp title=main.wasp
+app MyApp {
+  // ...
+  db: {
+    // ...
+    seeds: [
+      import { devSeedSimple } from "@server/dbSeeds.js",
+      import { prodSeed } from "@server/dbSeeds.js"
+    ]
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+Each seed function must be an async function that takes one argument, `prismaClient`, which is a [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client/crud) instance used to interact with the database.
+This is the same Prisma Client instance that Wasp uses internally and thus includes all of the usual features (e.g., password hashing).
+
+Since a seed function falls under server-side code, it can import other server-side functions. This is convenient because you'll normally want to seed the database using Actions.
```

Comment:
```suggestion
Since a seed function falls under server-side code, it can import other server-side functions. This is convenient because you might want to seed the database using Actions.
```